gradio/_simple_templates/simpledropdown.py

```diff
@@ -38,7 +42,7 @@ def __init__(
             value: default value selected in dropdown. If None, no value is selected by default. If callable, the function will be called whenever the app loads to set the initial value of the component.
             label: component name in interface.
             info: additional component description.
-            every: If `value` is a callable, run the function 'every' number of seconds while the client connection is open. Has no effect otherwise. The event can be accessed (e.g. to cancel it) via this component's .load_event attribute.
+            every: If `value` is a callable, run the function every time the provided Timer ticks. Has no effect otherwise.
```

Comment:
If `value` is a callable, run the function 'every' number of seconds while the client connection is open. Has no effect otherwise. If a `Timer` component is provided, the the function will be run every time the provided Timer ticks.